### ISSUE TITLE
Definitions Missing

### ISSUE THREAD

**Issue by ben-norton** [DiSSCo/openDS#188]
### Suggestion

The following assumes that "definition" is being used as described in the SKOS specification (Supplies a complete explanation of the intended meaning of the type).
Several terms in the openDS are missing definitions or contain circular definitions where the term is used to define the term itself. Example
Term: ods:Assertion (class)
Definition: The Assertion of the digital specimen
This definition doesn't help me understand what assertion is meant to represent in this context.

**Comment by DavidFichtmueller** [comment]
Yes we also noticed this for several terms. I have created issues for those terms, where we actually had a problem to understand the intended meaning, but there are probably more.

**Comment by samleeflang** [comment]
Thanks for the feedback! We will go over all definitions and improve them, avoiding any circular definitions.
